react/__tests__/toastArea.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import { wait } from '@apollo/react-testing'
+import { render } from '@vtex/test-tools/react'
+import React from 'react'
+
+import ToastArea from '../Components/toastArea'
+import { ContextStore } from '../hooks/useStore'
+import { ShowAlertOptions } from '../utils/showAlertOptions'
+// eslint-disable-next-line jest/no-mocks-import
+import { values } from '../__mocks__/values'
+
+describe('Toast Area', () => {
+  it('should render success Alert if showAlert = 0', async () => {
+    const showAlert = { showAlert: ShowAlertOptions.notShow }
+
+    const { container } = render(
+      <ContextStore.Provider value={{ ...values, ...showAlert }}>
+        <ToastArea DEFAULT_ALERT_AUTOCLOSE_TIMEOUT_MS={3000} />
+      </ContextStore.Provider>
+    )
+
+    expect(container.firstChild).toBeNull()
+  })
+
+  it('should render success Alert if showAlert = 1', async () => {
+    const showAlert = { showAlert: ShowAlertOptions.alertSave }
+
+    const { container } = render(
+      <ContextStore.Provider value={{ ...values, ...showAlert }}>
+        <ToastArea DEFAULT_ALERT_AUTOCLOSE_TIMEOUT_MS={3000} />
```

Comment:
If when you use the component the time duration is the same of default value, set this value is not necessary.
```suggestion
        <ToastArea />
```